ESP3/buffer2struct.js

```diff
@@ -0,0 +1,16 @@
+module.exports = function (buf) {
+  var dl = parseInt(buf.slice(1, 3).toString('hex'), 16)
+  var odl = buf[3]
+  return {
```

Comment:
What do you think about validation(CRC8-Checks) before returning?
I think it will be useful(maybe not at this place, but somewhere earlier?), because sending invalid Packets would fail anyway.

Reply:
yeah, no error handling in place at all yet... maybe instead of checking we can recalculate the checksum here and overwrite  what was passed here, because that removes the need to calculate the crc before passing it here. making it easier to construct packets higher up the abstraction chain.